lib/plugins/filter/meta_generator.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Also, it seems new filter added here. But in pull readme there is nothing about this.

Reply:
That change is the result of # 3129 being resolved by running git rebase.